internal/log/log_test.go

```diff
@@ -21,134 +22,216 @@ import (
 	"testing"
 
 	"github.com/vdaas/vald/internal/errors"
-	"github.com/vdaas/vald/internal/log/format"
 	"github.com/vdaas/vald/internal/log/glg"
 	"github.com/vdaas/vald/internal/log/level"
-	logger "github.com/vdaas/vald/internal/log/logger"
+	"github.com/vdaas/vald/internal/log/logger"
 	"github.com/vdaas/vald/internal/log/mock"
-
 	"go.uber.org/goleak"
 )
 
+var (
+	// Goreaine leak is detected by `fastime`, but it should be ignored in the test because it is an external package.
```

Comment:
```suggestion
	// goroutine leak is detected by `fastime`, but it should be ignored in the test because it is an external package.
```
it seems goroutine not goreaine, right?

Reply:
thank you for your review. I will fix it soon.